api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -355,7 +357,11 @@ public GenomicExtractionJob submitGenomicExtractionJob(DbWorkspace workspace, Db
     // keeping overhead low and limiting footprint on shared extraction quota.
     int minScatter =
         Math.min(cohortExtractionConfig.minExtractionScatterTasks, MAX_EXTRACTION_SCATTER);
-    int scatter = Ints.constrainToRange(personIds.size() / 2, minScatter, MAX_EXTRACTION_SCATTER);
+    int scatter =
+        Ints.constrainToRange(
+            Math.round(personIds.size() * EXTRACTION_SCATTER_FACTOR),
```

Comment:
Can we make this `EXTRACTION_SCATTER_FACTOR` configurable to avoid having to make code changes to update it? Given that it's still experimental. 
Also maybe worth documenting that putting its value to 0.5 will change the behavior back to what it used to be...

Reply:
Done - we shouldn't overuse this vs constants, but in this case I think it's a good suggestion.

I'm not sure where you are suggesting I document that .5 was the previous behavior. Code revision history seems sufficient to me in this case.